app/assets/src/components/views/bulk_download/BulkDownloadModal.jsx

```diff
@@ -118,102 +98,65 @@ class BulkDownloadModal extends React.Component {
     return sampleValidationInfo;
   }
 
-  handleSelectDownloadType = selectedDownloadTypeName => {
-    this.setState({
-      selectedDownloadTypeName,
-    });
-  };
+  // TODO(mark): Set a reasonable default background based on the samples and the user's preferences.
+  async fetchBackgrounds() {
+    const backgrounds = await getBackgrounds();
 
-  handleFieldSelect = (downloadType, fieldType, value, displayName) => {
-    this.setState(prevState => {
-      // If the value is undefined, delete it from selectedFields.
-      // This allows us to support cases where certain fields are conditionally required;
-      // if the field becomes no longer required, we can unset it.
-      const newSelectedFields =
-        value !== undefined
-          ? set([downloadType, fieldType], value, prevState.selectedFields)
-          : unset([downloadType, fieldType], prevState.selectedFields);
-
-      const newSelectedFieldsDisplay =
-        displayName !== undefined
-          ? set(
-              [downloadType, fieldType],
-              displayName,
-              prevState.selectedFieldsDisplay
-            )
-          : unset([downloadType, fieldType], prevState.selectedFieldsDisplay);
-
-      return {
-        selectedFields: newSelectedFields,
-        selectedFieldsDisplay: newSelectedFieldsDisplay,
-      };
-    });
-  };
+    const backgroundOptions = backgrounds.map(background => ({
+      text: background.name,
+      value: background.id,
+    }));
 
-  handleChooseStepContinue = () => {
-    this.setState({ currentStep: "review" });
-  };
-  handleBackClick = () => {
-    this.setState({ currentStep: "choose" });
-  };
+    return backgroundOptions;
+  }
+
+  // We use the heatmap metrics as the valid metrics for bulk downloads.
```

Comment:
If we are using heatmap metrics more widely, we should probably move to a more generic endpoint.
I guess this was not introduced on this PR, so might not be in scope...